readme.markdown

```diff
@@ -15,10 +15,10 @@ var test = require('tape');
 
 test('timing test', function (t) {
     t.plan(2);
-    
```

Comment:
please revert unrelated whitespace changes.

Reply:
Ha, yeah, I was just looking into how to stop VS Code from adding those :)